setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="darwin-py",
-    version="0.0.3",
+    version="0.0.4",
```

Comment:
we don't need to increase until we have pushed it to pip.

Reply:
I manually import this in Gale for the moment hence the upgrade. 
Shall i roll back to 0.0.3 ? (or 0.0.2 ?)